kaitai_utils.py

```diff
@@ -37,16 +39,66 @@ def get_plugin_title_from_metadata():
         return _to_camel_case(metadata["id"])
 
 
-def write_to_json(data_binary: BinaryIO, writer: BufferedWriter, class_type: Type[KaitaiStruct]):
-    """
-    Writes a binary form of JSON string into a BufferedWriter
-
-    @param data_binary: binary data containing the file content
-    @param writer: bufferedWriter to write the binary form of the JSON string to
-    @param class_type: class that contains the parsing to a KaiTai struct
-    @return: JSON string representing contents of data object
-    """
-    writer.write(bytes(to_json_string(data_binary, class_type), "utf-8"))
+class JsonWriter:
```

Comment:
i would prefer the class JsonWriter to be in a separate file JsonWriter.py

Reply:
At the NFI, separate files for classes is considered as beging not very _pythonic_. (In Python it is common to have these things in a single file).
Given that this is a template, the user of the template is not going to inspect/update this, it would be nicer to have this in the same file (all 'boilerplate' is here).

If we really want a separate file, please move all the logic to a separate _module_, so all 'boilerplate' is grouped in a single folder.

Reply:
We discussed this in person and decided not to do it.